src/Core/Parsers/RequestParser.cs

```diff
@@ -30,7 +30,7 @@ public class RequestParser
         /// <summary>
         /// Prefix used for specifying limit in the query string of the URL.
         /// </summary>
-        public const string FIRST_URL = "$first";
+        public const string FIRST_URL = "$top";
```

Comment:
We will provide a synonym $top soon. Lets not make this change in this PR. Thank you for your patience.